test/test_dtype.py

```diff
@@ -97,9 +97,10 @@ def test_upcast_to_ops(self):
   def test_bitcast(self):
     if Device.DEFAULT == "WEBGL": raise unittest.SkipTest("no bitcast in WebGL GLSL")
     if self.DTYPE == dtypes.bool: raise unittest.SkipTest("no bools in bitcast")
+    if self.DTYPE in [dtypes.fp8_e4m3, dtypes.fp8_e5m2]: raise unittest.SkipTest("no bitcast test support for fp8")
```

Comment:
Moved it to the place where target as bfloat16 is avoided. It's not supported because of the same reason that bfloat16 target tests are currently not done (cannot use numpy to test the result for fp8s)